### DISCUSSION TITLE
How can I block certain Sections?

### DISCUSSION THREAD

**Discussion by alfkoblischke** [codeigniter4/shield#690] [Q&A] [answered]
I got Shield up and working. How can I block for instance my Controller "Help" for Users not logged in? There must be a global solution, without having to check for each function within the Controller.
Can anybody help?

**Comment by datamweb** [comment, marked as answer]
Hello, please see:
https://codeigniter4.github.io/shield/install/#protect-all-pages

If you want to protect a specific route, you can do the following:
```php
$routes->get('xxx', 'PayController::index', ['filter' => 'session']);
```

**Reply by alfkoblischke** [reply]
Perfect. It worked with the Routes Filter. Thank you very much!
